config/manifests-multitenant-rhoam/bases/multitenant-managed-api-service.clusterserviceversion.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    operatorframework.io/suggested-namespace: sandbox-rhoam-operator
+    alm-examples: '[]'
```

Comment:
Hey Carl, not sure where this warning is coming from 
```
WARN[0000] ClusterServiceVersion validation: [OperationFailed] provided API should have an example annotation 
```
But I guess it's because the newly generated CSV does not cover an example of tenants CRD